src/SkeletalTrapezoidation.cpp

```diff
@@ -522,25 +512,34 @@ void SkeletalTrapezoidation::collapseSmallEdges(coord_t snap_dist)
 {
     std::unordered_map<edge_t*, std::list<edge_t>::iterator> edge_locator;
     std::unordered_map<node_t*, std::list<node_t>::iterator> node_locator;
+    
     for (auto edge_it = graph.edges.begin(); edge_it != graph.edges.end(); ++edge_it)
+    {
         edge_locator.emplace(&*edge_it, edge_it);
-    for (auto node_it = graph.nodes.begin(); node_it != graph.nodes.end(); ++node_it )
-        node_locator.emplace(&*node_it, node_it );
+    }
+    
+    for (auto node_it = graph.nodes.begin(); node_it != graph.nodes.end(); ++node_it)
```

Comment:
Last time I used `auto` a certain someone that used to work for us was very vocal that it was _not_ in our code-style.

Reply:
But it's also not against our code style, right? 

What would be the reason to not use auto?

Reply:
I looked it up and it isn't against our code style.
A case of misremembering the original comments.

If I now look back I don't agree with my use of auto there either actually.
The way Tim uses them here is exactly what I think they should be used for however.